ping.js

```diff
@@ -30,3 +30,40 @@ Ping.host = function (host) {
     console.log('not a valid argument for ping');
   }
 };
+
+/**
+ * Ping a range of IPs.
+ * @param {array} IP list - array of IP addresses
+ * @returns {undefined}
+ */
+Ping.range = function (range){
+	var i, ipv4Regex = /^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/;
```

Comment:
Do you think it makes sense to move the regexp vars to the beginning of this file? Just to avoid duplication and it can be used in any function.